com.unity.cinemachine/Editor/Editors/InputAxisControllerEditor.cs

```diff
@@ -112,13 +71,14 @@ static DefaultControlInitializer()
 #pragma warning restore CS0219 // Variable is assigned but its value is never used
                     var inputName = "";
                     var invertY = false;
-                    controller.Recentering = InputAxisRecenteringSettings.Default;
+                    bool isMomentary = (axis.DrivenAxis().Restrictions & InputAxis.RestrictionFlags.Momentary) != 0;
 
                     if (axis.Name.Contains("Look"))
                     {
                         actionName = "Player/Look";
-                        inputName = axis.AxisIndex == 0 ? "Mouse X" : (axis.AxisIndex == 1 ? "Mouse Y" : "");
-                        invertY = axis.AxisIndex == 1;
+                        inputName = axis.Hint == IInputAxisSource.AxisDescriptor.Hints.X ? "Mouse X" 
+                            : (axis.Hint == IInputAxisSource.AxisDescriptor.Hints.Y ? "Mouse Y" : "");
```

Comment:
This can be written with this syntax, which is a bit clearer (but longer):
```
inputName = axis.Hint switch
{
    IInputAxisSource.AxisDescriptor.Hints.X => "Mouse X",
    IInputAxisSource.AxisDescriptor.Hints.Y => "Mouse Y",
    _ => ""
};  
```